source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -610,7 +610,7 @@ are not replica set member states at all.
 A server can transition from any state to any other.  For example, an
 administrator could shut down a secondary and bring up a mongos in its place.
 
-When a server transitions from Unknown to data-bearing type, the driver MUST
+When a server transitions to data-bearing type, the driver MUST
 ensure the server's connection pool is set up (if the driver implements
 connection pooling). See `Connection Pool Management`_ for more information.
```

Comment:
I think we can remove this paragraph altogether. It does not really fit in this "Parsing an ismaster response" section of the spec and the pool management behavior is already described in the "Connection Pool Management" section: https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#connection-pool-management

Reply:
done